### ISSUE TITLE
M122 R4 - HEVC/AC3/E-AC3 still broken

### ISSUE THREAD

**Issue by Prinz23** [Alex313031/thorium-legacy#97]
**System Details**
 - OS: Windows 8.1
 - Thorium AVX2 122.0.6261.171

**Problem**
HEVC/AC3/E-AC3 is still not working. Tested both with Jellyfin and even on your testing website it doesn't play: https://thorium.rocks/misc/h265-tester.html

Going back to an much older release it all works again.

**Comment by Alex313031** [comment]
Yeah. It wouldn't work on XP or 7. And now you are saying 8.1. It worked on 10 and 11.

So something is broken in the HEVC decoder or presentation code on oses earlier than 10.

What was the old version that you were able to get it to work on.

**Comment by Prinz23** [comment]
Currently using Version 109.0.5414.173 (Official Build) (64-bit) . Which works fine.
